fastify.js

```diff
@@ -256,6 +259,41 @@ function build (options) {
     return fastify
   }
 
+  // creates an iterator with all the routes
+  function exposeRoutes () {
+    var iterable = {}
+    iterable[Symbol.iterator] = function () {
```

Comment:
In this way the api is super nice, but we can iterate just one time over it.
Example:
```js
fastify.get('/', () => {})
...
for (var route of fastify) {
  // route['/'].get will exist
}
for (route of fastify) {
  // route['/'].get will NOT exist
}
```
I think because the internal state of the iterator. Do you know any workaround for this? Put it inside a function (like before) solved the problem for example.